File: sdks/thunkmetrc-ts/wrapper/src/models/ActiveJobType.ts

import type { ProcessingJobActiveJobTypeAttributesItem } from './ProcessingJobActiveJobTypeAttributesItem';
export interface ActiveJobType {
    Attributes?: ProcessingJobActiveJobTypeAttributesItem[];
    CategoryName?: string;
    Description?: string;
    ForItems?: boolean;
    ForProcessingJobs?: boolean;
    Id?: number;
    Name?: string;
    ProcessingSteps?: string;
    RequiresProcessingJobAttributes?: boolean;
}

File: sdks/thunkmetrc-py/wrapper/src/thunkmetrc/wrapper/models/active_job_type.py
from typing import TypedDict, List, Optional, Any, Generic, TypeVar, TYPE_CHECKING
from typing_extensions import TypedDict

if TYPE_CHECKING:
    from . import *

class ActiveJobType(TypedDict, total=False):
    Attributes: List['ProcessingJobActiveJobTypeAttributesItem']
    CategoryName: str
    Description: str
    ForItems: bool
    ForProcessingJobs: bool
    Id: int
    Name: str
    ProcessingSteps: str
    RequiresProcessingJobAttributes: bool
